### ISSUE TITLE
Create labels seems to work but do not add labels :s

### ISSUE THREAD

**Issue by MoOx** [popomore/github-labels#12]
```console
$ cat ~/.github-issues-labels      
[
  {"name": "bug", "color": "ffffff"},
  {"name": "feature", "color": "000000"}
]

$ labels -c ~/.github-issues-labels stylelint/stylelint -f
>> Authorized
>> Delete existing labels
>> Create labels
>> Done

$ labels -c ~/.github-issues-labels stylelint/stylelint   
>> Authorized
>> Create labels
>> Done
```

And I get nothing here https://github.com/stylelint/stylelint/labels

What is wrong ?

**Comment by popomore** [comment]
I don't know where is the problem, I tried your config in my repo https://github.com/popomore/test/labels

**Comment by popomore** [comment]
Should log the label

```
./bin/labels -c ./a.json popomore/test -f
>> Authorized
>> Delete existing labels
>> Create labels
>>   Create label bug
>>   Create label feature
>> Done
```

**Comment by MoOx** [comment]
Sounds good to do that for a start :)

(Note that `-f` (delete)  worked on this repo)

**Comment by MoOx** [comment]
Maybe it is because it's on an org, not on a user repo ? Can this be related ?

**Comment by popomore** [comment]
Maybe, I have not test it on org.

**Comment by popomore** [comment]
Also worked on org, see https://github.com/test-labels/test/labels

```
./bin/labels -c ./a.json test-labels/test -f 
>> Authorized
>> Delete existing labels
>> Create labels
>>   Create label bug
>>   Create label feature
>> Done
```

**Comment by MoOx** [comment]
:sob:

**Comment by MoOx** [comment]
I will try to investigate.

**Comment by popomore** [comment]
You can compare `old` and `label`

https://github.com/popomore/github-labels/blob/master/lib/label.js#L29

**Comment by popomore** [comment]
Is it fixed?

**Comment by MoOx** [comment]
Ok I just make some tests and it seems it only works with `.json` files. If you use a json file wich does not have .json extension, it just provide a configs set to `null`.

**Comment by MoOx** [comment]
Probably related to how commander works.

**Comment by MoOx** [comment]
Hum it's more how you handle config file in your index.js

**Comment by popomore** [comment]
I'll look at it.

**Comment by popomore** [comment]
I have released 0.5.2, you can have a try.
